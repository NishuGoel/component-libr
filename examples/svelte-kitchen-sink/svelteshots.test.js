import path from 'path';
import initStoryshots, { multiSnapshotWithOptions } from '@storybook/addon-storyshots';

initStoryshots({
  framework: 'svelte',
  configPath: path.join(__dirname, '.storybook'),
  integrityOptions: { cwd: path.join(__dirname, 'src', 'stories') },
  test: multiSnapshotWithOptions(),
});
