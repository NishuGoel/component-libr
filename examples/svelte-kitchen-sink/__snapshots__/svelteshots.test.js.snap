// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Storyshots Addon|Actions Action on component method 1`] = `
<section
  class="storybook-snapshot-container"
>
  <button
    class="button svelte-n2tx93 rounded"
  >
    <strong>
      Round
       corners
    </strong>
    <br />
     
    Custom text
     
  </button>
</section>
`;

exports[`Storyshots Addon|Actions Action on view method 1`] = `
<section
  class="storybook-snapshot-container"
>
  <h1>
    Button view
  </h1>
   
  <button
    class="button svelte-n2tx93"
  >
    <strong>
      Square
       corners
    </strong>
    <br />
     
    
     
    You clicked
    : 
    0
  </button>
   
  <p>
    A little text to show this is a view.
  </p>
   
  <p>
    If we need to test components in a Svelte environment, for instance to test slot behaviour,
  </p>
   
  <p>
    then wrapping the component up in a view
  </p>
   
  <p>
    made just for the story is the simplest way to achieve this.
  </p>
</section>
`;

exports[`Storyshots Addon|Backgrounds story 1 1`] = `
<section
  class="storybook-snapshot-container"
>
  <h1>
    Button view
  </h1>
   
  <button
    class="button svelte-n2tx93"
  >
    <strong>
      Square
       corners
    </strong>
    <br />
     
    
     
    You clicked
    : 
    0
  </button>
   
  <p>
    A little text to show this is a view.
  </p>
   
  <p>
    If we need to test components in a Svelte environment, for instance to test slot behaviour,
  </p>
   
  <p>
    then wrapping the component up in a view
  </p>
   
  <p>
    made just for the story is the simplest way to achieve this.
  </p>
</section>
`;

exports[`Storyshots Addon|Centered rounded 1`] = `
<section
  class="storybook-snapshot-container"
>
  <button
    class="button svelte-n2tx93 rounded"
  >
    <strong>
      Round
       corners
    </strong>
    <br />
     
    Look, I'm centered!
     
  </button>
</section>
`;

exports[`Storyshots Addon|Centered with action 1`] = `
<section
  class="storybook-snapshot-container"
>
  <button
    class="button svelte-n2tx93 rounded"
  >
    <strong>
      Round
       corners
    </strong>
    <br />
     
    
     
  </button>
</section>
`;

exports[`Storyshots Addon|Knobs Simple 1`] = `
<section
  class="storybook-snapshot-container"
>
  <div
    style="width: 200px; height: 200px; background-color: green;"
  >
    <p>
      I am interactive
    </p>
  </div>
</section>
`;

exports[`Storyshots Addon|Links Go to welcome view 1`] = `
<section
  class="storybook-snapshot-container"
>
  <div
    class="main"
  >
    <h1>
      Link Action
    </h1>
     
    <button
      class="link"
    >
      
			    Return to the 
      <code
        class="code"
      >
        Welcome
      </code>
       view story.
			  
    </button>
  </div>
</section>
`;

exports[`Storyshots Addon|Notes Note with HTML 1`] = `
<section
  class="storybook-snapshot-container"
>
  <h1>
    Button view
  </h1>
   
  <button
    class="button svelte-n2tx93"
  >
    <strong>
      Square
       corners
    </strong>
    <br />
     
    
     
    ðŸ¤”ðŸ˜³ðŸ˜¯ðŸ˜®ðŸ˜„ðŸ˜©ðŸ˜“ðŸ˜±ðŸ¤“ðŸ˜‘ðŸ˜¶ðŸ˜Š
    : 
    0
  </button>
   
  <p>
    A little text to show this is a view.
  </p>
   
  <p>
    If we need to test components in a Svelte environment, for instance to test slot behaviour,
  </p>
   
  <p>
    then wrapping the component up in a view
  </p>
   
  <p>
    made just for the story is the simplest way to achieve this.
  </p>
</section>
`;

exports[`Storyshots Addon|Notes Simple note 1`] = `
<section
  class="storybook-snapshot-container"
>
  <h1>
    Button view
  </h1>
   
  <button
    class="button svelte-n2tx93"
  >
    <strong>
      Square
       corners
    </strong>
    <br />
     
    
     
    You clicked
    : 
    0
  </button>
   
  <p>
    A little text to show this is a view.
  </p>
   
  <p>
    If we need to test components in a Svelte environment, for instance to test slot behaviour,
  </p>
   
  <p>
    then wrapping the component up in a view
  </p>
   
  <p>
    made just for the story is the simplest way to achieve this.
  </p>
</section>
`;

exports[`Storyshots Button rounded 1`] = `
<section
  class="storybook-snapshot-container"
>
  <h1>
    Button view
  </h1>
   
  <button
    class="button svelte-n2tx93 rounded"
  >
    <strong>
      Round
       corners
    </strong>
    <br />
     
    
     
    You clicked
    : 
    0
  </button>
   
  <p>
    A little text to show this is a view.
  </p>
   
  <p>
    If we need to test components in a Svelte environment, for instance to test slot behaviour,
  </p>
   
  <p>
    then wrapping the component up in a view
  </p>
   
  <p>
    made just for the story is the simplest way to achieve this.
  </p>
</section>
`;

exports[`Storyshots Button square 1`] = `
<section
  class="storybook-snapshot-container"
>
  <h1>
    Button view
  </h1>
   
  <button
    class="button svelte-n2tx93"
  >
    <strong>
      Square
       corners
    </strong>
    <br />
     
    
     
    You clicked
    : 
    0
  </button>
   
  <p>
    A little text to show this is a view.
  </p>
   
  <p>
    If we need to test components in a Svelte environment, for instance to test slot behaviour,
  </p>
   
  <p>
    then wrapping the component up in a view
  </p>
   
  <p>
    made just for the story is the simplest way to achieve this.
  </p>
</section>
`;

exports[`Storyshots Welcome Welcome 1`] = `
<section
  class="storybook-snapshot-container"
>
  <div
    class="main svelte-5n9a95"
  >
    <h1>
      Welcome to Storybook for Svelte
    </h1>
     
    <p>
      
			    This is a UI component dev environment for your svelte app.
			  
    </p>
     
    <p>
      
			    We've added some basic stories inside the 
      <code
        class="code svelte-5n9a95"
      >
        src/stories
      </code>
       directory.
			    
      <strong>
        A story is like a visual test case
      </strong>
      
			    and represents a single state of one or more UI components.
			    You can have as many stories as you want.
			  
    </p>
     
    <h1
      class="logo svelte-5n9a95"
    >
      Svelte
    </h1>
     
    <p>
      
			    Just like that, you can add your own components as stories.
			    You can also edit those components and see changes right away.
			    
      <br />
      
			    (Try editing the 
      <code
        class="code svelte-5n9a95"
      >
        Button
      </code>
       component
			    located at 
      <code
        class="code svelte-5n9a95"
      >
        src/stories/views/Welcome.svelte
      </code>
      .)
			  
    </p>
     
    <p>
      
			    Usually we create stories with smaller UI components in the app.
      <br />
      
			    Have a look at the
			    
      <a
        class="link svelte-5n9a95"
        href="https://storybook.js.org/basics/writing-stories"
        target="_blank"
      >
        Writing Stories
      </a>
      
			    section in our documentation.
			  
    </p>
     
    <p
      class="note svelte-5n9a95"
    >
      <b>
        NOTE:
      </b>
       
      <br />
      
			    Have a look at the
			    
      <code
        class="code svelte-5n9a95"
      >
        .storybook/webpack.config.js
      </code>
      
			    to add webpack
			    loaders and plugins you are using in this project.
			  
    </p>
  </div>
</section>
`;
